### PR TITLE
update mobile doc

### PR DESCRIPTION
**Pull request by shanyi15**
Update according to https://github.com/PaddlePaddle/paddle-mobile/blob/develop/README.md

### CODE REVIEW

File: doc/fluid/advanced_usage/deploy/mobile_readme.md [hunk 1]
```diff
@@ -1,125 +1,80 @@
...
+
+* [iOS](https://github.com/PaddlePaddle/paddle-mobile/blob/develop/doc/development_ios.md)
+
+* [Android](https://github.com/PaddlePaddle/paddle-mobile/blob/develop/doc/development_android.md)
+
+* [FPGA](https://github.com/PaddlePaddle/paddle-mobile/blob/develop/doc/development_fpga.md)
+
+* [ARM_LINUX](https://github.com/PaddlePaddle/paddle-mobile/blob/develop/doc/development_arm_linux.md)
 
 ### 贡献文档
```
Comment (tink2123) [inline doc/fluid/advanced_usage/deploy/mobile_readme.md]: 贡献代码？

File: doc/fluid/advanced_usage/deploy/mobile_readme.md [hunk 2]
```diff
@@ -1,125 +1,80 @@
...
+目前Paddle-Mobile仅支持Paddle fluid训练的模型。如果你手中的模型是不同种类的模型，需要进行模型转换才可以运行
+
 ### 1. 直接使用Paddle Fluid训练
+
 该方式最为可靠，推荐方式
+
 ### 2. Caffe转为Paddle Fluid模型
-[链接](https://github.com/PaddlePaddle/models/tree/develop/fluid/image_classification/caffe2fluid)
+
+请参考这里[这里](https://github.com/PaddlePaddle/models/tree/develop/fluid/image_classification/caffe2fluid)
```
Comment (tink2123) [inline doc/fluid/advanced_usage/deploy/mobile_readme.md]: not found

**Comment by shanyi15** [comment]
Done

**Review by tink2123 — APPROVED** [review]
LGTM

**Pull request merged**